src/main/java/ru/naumen/servacc/ui/UIController.java

```diff
@@ -520,7 +494,6 @@ private void createEncryptMenuItem(final Menu menu)
         itemEncrypt.setText("Encrypt Local Accounts");
         itemEncrypt.addListener(SWT.Selection, new Listener()
         {
-            @Override
             public void handleEvent(Event event)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1161)